android/build.gradle

```diff
@@ -1,5 +1,7 @@
 apply plugin: 'com.android.library'
 
+group = "io.grpc"
+version = "1.13.0-SNAPSHOT" // CURRENT_GRPC_VERSION
```

Comment:
I think the versions should be changed to 1.12.0-SNAPSHOT  here and below

Reply:
Done, thanks